dt/cloudops/openstack-telemetry_DT.md

```diff
@@ -0,0 +1,54 @@
+# RHOSO Deployed Topology OpenStack Observability
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/tree/78b3c876eaf9168f9d95b201997ebdc2da42fa02) on Oct 17th, 2023**
+
+## General information
+
+| Revision | Change                | Date             |
+|--------: | :-------------------- | :--------------: |
+| v0.1     | Initial publication   | 14.12.23      |
+
+## Node topology
+| Node role                                     | bm/vm | amount |
+| --------------------------------------------- | ----- | ------ |
+| Compute nodes                                 | vm    | 2      |
+| Controller nodes                              | vm    | 3      |
+
+
+## Services, enabled features and configurations
+| Service                                     | configuration                     | Lock-in coverage?  |
+| ------------------------------------------- | ----------------------------------| ------------------ |
+| Nova                                        | default                           | Standard           |
+| Neutron                                     | default                           | Standard           |
+| Cinder                                      | default                           | Standard           |
+| Glance                                      | default                           | Standard           |
+| Keystone                                    | default                           | Standard           |
```

Comment:
I dont know if Keystone already deploys them, but Telemetry depends on Memcached and RabbitMQ.

We should define Memcached, RabbitMQ and Keystone as Must have due to prerequisites.